src/dev/run_with_node_options/run_with_node_options_cli.js

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
CLI specific settings seems kind of confusing to me since everything is technically ran through the CLI. Seems like it might be an over optimization at the moment. Thoughts?

Reply:
I think if we decide to go with the strategy described above we can just let go that CLI or do you think it might be some situation or that could still be useful?